domains/anubhav-pathak.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "Documentation for is-a-dev",
+    "repo": "https://github.com/Anubhav-Pathak/Anubhav-Pathak.github.io.git",
+    "owner": {
+      "username": "Anubhav-Pathak",
+      "email": "anubhavpathak99@gmail.com"
+    },
+    "record": {
+      "CNAME": "anubhav-pathak.github.io",  
+      "TXT": "af5fdf5f2c55b83aade6d80e125efd"
```

You can't have CNAME with any records, since I think TXT is for github pages verification go here to find the steps on how to make it https://www.is-a.dev/docs/github-pages-verification/, also site seems to redirect to is-a.dev so could you disconnect the domain (you can delete CNAME file in the websites repo) so that we can see your website?
```suggestion
      "CNAME": "anubhav-pathak.github.io"
```

---

I have removed the TXT field from the records and the CNAME from my website.